doc/man3/SSL_CTX_set_client_hello_cb.pod

```diff
@@ -56,7 +57,6 @@ pointer to the octets of that field.
 Similarly, SSL_client_hello_get0_ext() provides access to individual extensions
 from the ClientHello on a per-extension basis.  For the provided wire
 protocol extension type value, the extension value and length are returned
-in the output parameters (if present).
```

Comment:
This is wrongly dropped here and inserted below.

Reply:
done